usr/sbin/truenas-install

```diff
@@ -421,7 +421,6 @@ partition_disks()
     fi
 
     _disksparts=$(for _disk in ${_disks}; do
-	create_partitions ${_disk} >&2
```

Comment:
I am not sure what is happening here, but a few lines above we've ran `sgdisk -Z /dev/${_disk}` for all disks twice, won't it destroy all partitions?